net/stubby/files/stubby.yml

```diff
@@ -33,6 +41,14 @@ upstream_recursive_servers:
 # # Cloudflare servers
   - address_data: 1.1.1.1
     tls_auth_name: "cloudflare-dns.com"
+    tls_pubkey_pinset:
```

Comment:
From what I have read, it is not currently recommended to do this with the cloudflare service.

https://dnsprivacy.org/wiki/display/DP/DNS+Privacy+Public+Resolvers